pyls/plugins/jedi_completion.py

```diff
@@ -138,9 +140,9 @@ def _format_completion(d, include_params=True):
         path = path.replace('/', '\\/')
         completion['insertText'] = path
 
-    if (include_params and hasattr(d, 'params') and d.params and
-            not is_exception_class(d.name)):
-        positional_args = [param for param in d.params
+    sig = d.get_signatures()
```

Comment:
Why was the

`hasattr(d, 'params')`no longer needed?

Reply:
`get_signatures` always returns a list of `BaseSignature`s, which always have the attribute `params` (see https://jedi.readthedocs.io/en/latest/docs/api-classes.html?highlight=get_signatures#jedi.api.classes.BaseName.get_signatures)

Reply:
Awesome, thanks!